pysand/erosion.py

```diff
@@ -40,6 +40,10 @@ def validate_inputs(**kwargs):
                 logger.warning('The model has got negative value(s) of {} and returned nan.'.format(i))
                 return True
 
+    if 'crushed' in kwargs:
+        if not isinstance(kwargs['crushed'], bool):
+            raise exc.FunctionInputFail('{} is not a boolean'.format('crushed'))
```

Comment:
use the f-strings Luke!

Reply:
You could also write a test for this exception?

Reply:
> You could also write a test for this exception?

I thought I did. Line 21-26 in test_erosion.py?

Reply:
Ah yes, good work :)